src/decoding.rs

```diff
@@ -168,7 +168,7 @@ pub struct ParsedModRM{
 impl ParsedModRM{
     /// Decodes the current byte stream (starting at the ModRM byte) into a fully formed ParsedModRM struct
     pub fn from_bytes(bytestream: &[u8]) -> Result<ParsedModRM, VMError>{
-        if bytestream.len() < 16{
+        if bytestream.len() < 5{
```

Comment:
why disconnect between this and the len change below?

Reply:
The Mod R/M data can be at most 5 bytes large, while the decode_args_with_modrm stuff also includes the opcode byte, so can be at most 6 bytes large in that context